### PR TITLE
Added required arguments to the policy builder in the ACER model to

### PR DESCRIPTION
**Pull request by ethanwaldie**
fix issue #783

### CODE REVIEW

**Comment by pzhokhov** [comment]
@ethanwaldie you mentioned in the issues that with the change you get warnings about early stopping? What kind of warnings?

**Comment by ethanwaldie** [comment]
HI @pzhokhov I realized I was not using the make_vec function from the runners.py file, and that this function suppresses early stopping warnings by default. I switched to this and the warning are now gone.

**Comment by pzhokhov** [comment]
Ok cool. Please address the comment, and I'll merge it.

**Comment by ethanwaldie** [comment]
> Ok cool. Please address the comment, and I'll merge it.

Done, Cheers.

File: baselines/acer/acer.py [hunk 1]
```diff
@@ -75,8 +75,8 @@ def __init__(self, policy, ob_space, ac_space, nenvs, nsteps, ent_coef, q_coef,
         train_ob_placeholder = tf.placeholder(dtype=ob_space.dtype, shape=(nenvs*(nsteps+1),) + ob_space.shape)
         with tf.variable_scope('acer_model', reuse=tf.AUTO_REUSE):
 
-            step_model = policy(observ_placeholder=step_ob_placeholder, sess=sess)
-            train_model = policy(observ_placeholder=train_ob_placeholder, sess=sess)
+            step_model = policy(nbatch=nenvs, nsteps=nsteps, observ_placeholder=step_ob_placeholder, sess=sess)
```
Comment (pzhokhov) [inline baselines/acer/acer.py]: Almost agree :) nsteps should be 1

Comment (ethanwaldie) [reply baselines/acer/acer.py]: Whoops! :) sorry I missed that, just updated!

**Pull request merged**